scalr/eval_and_analysis_pipeline.py

```diff
@@ -152,12 +155,15 @@ def gene_analysis(self):
         top_features = gene_analyser.top_feature_extraction()
         self.primary_analysis['top_features'] = top_features
 
-    def perform_downstream_anlaysis(self):
+    def full_samples_downstream_anlaysis(self):
         """A function to perform all downstream analysis tasks on model and data."""
```

Comment:
WE are not using model. right?

Reply:
correct. Model will not be used (or even passed) when using full_samples_analysis

Reply:
Then we should change the doc-string